sync2jira/downstream_issue.py

```diff
@@ -960,12 +960,24 @@ def _update_github_project_fields(client, existing, issue, github_project_fields
     """
 
     for name, values in github_project_fields.items():
-        _, jirafieldname = values
-        try:
-            existing.update({jirafieldname: str(getattr(issue, name))})
-        except JIRAError:
-            # Add a comment to indicate there was an issue
-            client.add_comment(existing, f"Error updating GitHub project field")
+        fieldvalue = str(getattr(issue, name))
+        if not fieldvalue:
+            continue
+        if name == 'storypoints':
+            _, jirafieldname = tuple(values['fieldmap'].items())[0]
+            try:
+                existing.update({jirafieldname: fieldvalue})
+            except JIRAError:
+                # Add a comment to indicate there was an issue
+                client.add_comment(existing, f"Error updating GitHub project storypoints field")
+        elif name == 'priority':
+            jira_priority = github_project_fields['priority']['options'][fieldvalue]
+            try:
+                existing.update({'priority': {'name': jira_priority}})
+            except JIRAError:
+                # Add a comment to indicate there was an issue
+                client.add_comment(existing, f"Error updating GitHub project priority field")
+            pass
```

Comment:
The cascading `if` "waterfall" structure is disappointing.  If we construct the `github_project_fields` data structure correctly, we should be able to do straightforward key-lookups instead of a series of `if` tests.